### DISCUSSION TITLE
Added New Features [To Do, Shuffling Pictures, Quotes]

### DISCUSSION THREAD

**Discussion by hendryong** [zombieFox/nightTab#348] [Ideas]
Hi guys! I'm new in this Github Community. 
I currently use NightTab as my default new tab page. I really love the customization and that NightTab offers.
Before using NightTab, i was using **Tabliss** as my default new tab page. There's some features in **Tabliss** that i found useful and **still missing** in NightTab.
Here are the features:
1. To Do - It would be nice if it can fit a Kanban board too
2. Quotes - Daily Motivational Quotes
3. Shuffling Pictures - Gives a new vibes by changing background every (day, 5 minutes, tab) and we can choose our preferred Picture Categories (eg: Mountains, City, Urban, Cars, etc.)

**Comment by metruzanca** [comment]
These are all things I've been wanting to do and I've got in mind a nice implementation to support all kinds of "widgets". Unfortunately, I don't have the time for it at this point in time.

Previously a big blocker for nighttab is we wanted to keep it "no permissions" however, we're experimenting with optional permissions and prompting userts for additional permissions, so as soon as either @zombieFox or myself find the time this might get some progress.

PS: the kanban board is probably not going to happen due to the fact that its not going to fit any of our tile size and adding more would have to require some serious rework of the grid system. However, using a large square tile we can accomodate something like a todolist and make it scroll. Quotes can be part of the header and the background is quite doable with [unsplash.](https://unsplash.com/)
